### DISCUSSION TITLE
Truncated text on multiplayer/playlist beatmap cards isn't using truncation indicator

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15981] [Q&A]
When beatmap metadata is longer than beatmap card size, the text is cut abruptly, instead of displaying truncation indicator "`...`":
![image](https://user-images.githubusercontent.com/53346335/145058299-4a75d35f-f97a-4c97-927a-ca59b4d81ffb.png)
![image](https://user-images.githubusercontent.com/53346335/145058339-33afe7bc-ad3c-4bc9-b10c-fa300dcc197d.png)

**Comment by bdach** [comment]
this was a temporary intentional decision, as the choice was between cut off text or wrong clickable area of the text. this will either be changed by an upcoming redesign, or fill flow will one day support ellipsis properly.

**Reply by Sinamer** [reply]
Should this discussion be marked as answered, since this is known, or left around until a solution for it is in the game?

**Reply by frenzibyte** [reply]
This can be kept as there's no other issue/discussion tracking this.
